grammars/css.cson

```diff
@@ -333,7 +333,11 @@
         'captures':
           '1':
             'name': 'keyword.other.unit.css'
-        'match': '(?x)\n\t\t\t\t\t    (?:-|\\+)?(?:(?:[0-9]+(?:\\.[0-9]+)?)|(?:\\.[0-9]+))\n\t\t\t\t\t    ((?:px|pt|ch|cm|mm|in|r?em|ex|pc|deg|g?rad|dpi|dpcm|dppx|fr|ms|s|turn|vh|vmax|vmin|vw)\\b|%)?\n\t\t\t\t\t'
+        # NOTE by @burodepeper
+        # I have no idea what the stuff in the regex was intended to do that I removed, but it doesn't seem to break anything either way. I've left the original here, just in case.
+        # Tests are still fine, and a manual inspection of a rather large css file didn't show anything gone bad.
+        # 'match': '(?x)\n\t\t\t\t\t    (?:-|\\+)?(?:(?:[0-9]+(?:\\.[0-9]+)?)|(?:\\.[0-9]+))\n\t\t\t\t\t    ((?:px|pt|ch|cm|mm|in|r?em|ex|pc|deg|g?rad|dpi|dpcm|dppx|fr|ms|s|turn|vh|vmax|vmin|vw)\\b|%)?\n\t\t\t\t\t'
```

Comment:
Yes, you can remove them. But it would be better if you replace escape sequences with tabs and whitespaces for readability (like it was before converting).